tests/manager/celery/tasks/test_playtime_entries.py

```diff
@@ -722,13 +733,16 @@ def mock_create_file(
         generate_playtime_report.delay().wait()
 
         # Assert
-        assert len(output_data) == 2
-        [(ds_a_filename, ds_a_data), (ds_b_filename, ds_b_data)] = [
-            (k, [r for r in csv.reader(StringIO(v))]) for k, v in output_data.items()
-        ]
+        assert len(output_data) == 3
+        [
+            (ds_a_filename, ds_a_data),
+            (ds_b_filename, ds_b_data),
+            (ds_c_filename, ds_c_data),
+        ] = [(k, [r for r in csv.reader(StringIO(v))]) for k, v in output_data.items()]
```

Comment:
Minor: I think the RHS could be simplified by replacing the inner list comprehension with the `list` function:

```suggestion
        ] = [(k, list(csv.reader(StringIO(v)))) for k, v in output_data.items()]
```